spec/longest_word_spec.rb

```diff
@@ -5,4 +5,8 @@
   it 'should return correct results' do
     expect(longest_word("Here is the longest word")).to eq "longest"
   end
+
+  it 'should return last longest word' do
+    expect(longest_word("I have sinned and have fallen short")).to eq "fallen"
```

Comment:
Have you heard of the four phases of testing, setup, exercise and verify and teardown?

Your spec shows all of that apart from the teardown which is not bad as the teardown is usually handled by database cleaner. However, the 3 phases here are not explicit. You should have a line between each phase to make them explicit and easier to read. Something like:


```
word = longest_word("I have sinned and have fallen short")

expect(word).to eq("fallen)
```